src/index.d.ts

```diff
@@ -4016,8 +4016,9 @@ declare namespace admin.messaging {
     sticky?: boolean;
 
     /**
-     * Set the time that the event in the notification occurred. Notifications in the 
-     * panel are sorted by this time.
+     * For notifications that inform users about events with an absolute time reference, sets
+     * the time that the event in the notification occurred in milliseconds. Notifications
```

Comment:
This is not really a milliseconds timestamp in this API. Might want to drop that part.